src/content/self-hosted/install/releases.mdx

```diff
@@ -5,6 +5,33 @@ sidebar_label: Release Notes
 id: releases
 ---
 
+## 1.14.0
+
+[DATE]
+
+This version is compatible with Kubernetes versions 1.24 to 1.27.
+
+### Deprecatation Notice
+
+- As of this versions deploying helm charts through the Okteto UI has been deprecated.
+- Private Registry credentials defined in helm has been deprecated. Private Registry Credentials are now supported natively by Okteto and credentials defined in helm will be automatically migrated. More info [here](ADD_LINK)
+
+### Bug Fixes
```

Comment:
We are using **Bugfixes** in other versions

Reply:
Yes, and we have to change the one for the `1.13` release to match also with `Bugfixes`